apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-1010-schema-changes/schema_constraints_drop_primary_keys.sql

```diff
@@ -0,0 +1,69 @@
+--
+alter table only cmac.beneficiaries
+drop constraint if exists beneficiaries_pkey;
+--
+alter table only cmac.beneficiaries_history
+drop constraint if exists beneficiaries_history_pkey;
+--
+alter table only cmac.beneficiaries_history_invalid_beneficiaries
+drop constraint if exists beneficiaries_history_invalid_beneficiaries_pkey;
+--
+alter table only cmac.beneficiary_monthly
+drop constraint if exists beneficiary_monthly_pkey;
+--
+alter table only cmac.carrier_claim_lines
+drop constraint if exists carrier_claim_lines_pkey;
+--
+alter table only cmac.carrier_claims
+drop constraint if exists carrier_claims_pkey;
+--
+alter table only cmac.dme_claim_lines
+drop constraint if exists dme_claim_lines_pkey;
+--
+alter table only cmac.dme_claims
+drop constraint if exists dme_claims_pkey;
+--
+alter table only cmac.hha_claim_lines
+drop constraint if exists hha_claim_lines_pkey;
+--
+alter table only cmac.hha_claims
+drop constraint if exists hha_claims_pkey;
+--
+alter table only cmac.hospice_claim_lines
+drop constraint if exists hospice_claim_lines_pkey;
+--
+alter table only cmac.hospice_claims
+drop constraint if exists hospice_claims_pkey;
+--
+alter table only cmac.inpatient_claim_lines
+drop constraint if exists inpatient_claim_lines_pkey;
+--
+alter table only cmac.inpatient_claims
+drop constraint if exists inpatient_claims_pkey;
+--
+alter table only cmac.loaded_batches
+drop constraint if exists loaded_batches_pkey;
+--
+alter table only cmac.loaded_files
+drop constraint if exists loaded_files_pkey;
+--
+alter table only cmac.medicare_beneficiaryid_history
+drop constraint if exists medicare_beneficiaryid_history_pkey;
+--
+alter table only cmac.medicare_beneficiaryid_history_invalid_beneficiaries
+drop constraint if exists medicare_beneficiaryid_history_invalid_beneficiaries_pkey;
+--
+alter table only cmac.outpatient_claim_lines
+drop constraint if exists outpatient_claim_lines_pkey;
+--
+alter table only cmac.outpatient_claims
+drop constraint if exists outpatient_claims_pkey;
+--
+alter table only cmac.partd_events
+drop constraint if exists partd_events_pkey;
+--
+alter table only cmac.snf_claim_lines
+drop constraint if exists snf_claim_lines_pkey;
+--
+alter table only cmac.snf_claims
+drop constraint if exists snf_claims_pkey;
```

Comment:
Why do we need to drop the PK indices? If it's just for speed, I'd personally rather pay the performance cost of having them, just to keep the migrations simpler and avoid oopses.

For similar reasons, I'd probably put _all_ of the indices that we will want in place up-front.

Reply:
Karl; thx for the review. I was mainly concerned with column/table names and data types. Hence the 'draft' status. A lot of the scripts were for my use (i.e., tearing down tables, playing with some constraints, etc.) so while those types of files are there, they won't be part of the real deal. 

If the table/column names are OK and the data types, then I'll start on the Flyaway script(s).

As for the schema, yea, I just did that so I would munge anything on my localhost by accident; the fact that the table names are (will be changed) allows us to put everything into public without name collisions which is fine by me.